### PR TITLE
Add Origin Trial token for Subgroups in localhost:8080

### PR DESCRIPTION
**Pull request by dneto0**
Adds a localhost:8080 token to standalone/index.html for local testing.

https://crbug.com/358117283

Followup to #3895

Issue: #

<hr>

**Requirements for PR author:**

- [x] All missing test coverage is tracked with "TODO" or `.unimplemented()`.
- [x] New helpers are `/** documented */` and new helper files are found in `helper_index.txt`.
- [x] Test behaves as expected in a WebGPU implementation. (If not passing, explain above.)
- [x] Test have be tested with compatibility mode validation enabled and behave as expected. (If not passing, explain above.)

**Requirements for [reviewer sign-off](https://github.com/gpuweb/cts/blob/main/docs/reviews.md):**

- [ ] Tests are properly located in the test tree.
- [ ] [Test descriptions](https://github.com/gpuweb/cts/blob/main/docs/intro/plans.md) allow a reader to "read only the test plans and evaluate coverage completeness", and accurately reflect the test code.
- [ ] Tests provide complete coverage (including validation control cases). **Missing coverage MUST be covered by TODOs.**
- [ ] Helpers and types promote readability and maintainability.

When landing this PR, be sure to make any necessary issue status updates.

### CODE REVIEW

**Review by kainino0x — COMMENTED** [review]
Previously we've served up the localhost tokens from `dev_server.ts` instead. Do you need this because you're running this through some other server on localhost?

If so, I think it's redundant to have it in both, so we could have it only here and not in `dev_server.ts`.

File: standalone/index.html [hunk 1]
```diff
@@ -15,6 +15,8 @@
     <meta name="viewport" content="width=device-width" />
     <!-- Chrome "WebGPU Subgroups Features" origin trial token for https://gpuweb.github.io (see dev_server.ts for localhost tokens) -->
```
Comment (kainino0x) [inline standalone/index.html:16]: Also be sure to update this comment
```suggestion
    <!-- Chrome "WebGPU Subgroups Features" origin trial token for https://gpuweb.github.io -->
```

**Review by kainino0x — APPROVED** [review]
Realized that other servers _are_ supposed to work when building to disk (`out/` directory) so it makes sense to support localhost:8080 on those.

We can remove the token from `dev_server.ts` (though it's not critical). If you do, update the comment below

File: standalone/index.html (hunk 1 above)
Comment (dneto0) [reply standalone/index.html]: yep, done.

**Review by kainino0x — APPROVED** [review]

**Pull request merged**